package.json

```diff
@@ -1,45 +1,84 @@
 {
-    "name": "@energyweb/iam-roles",
-    "version": "1.0.0",
-    "publishConfig": {
-      "access": "public",
-      "registry": "https://registry.npmjs.org"
-    },
-    "description": "Smart contract and client code specific to EnergyWeb IAM roles",
-    "main": "dist/index.js",
-    "module": "dist/index.esm.js",
-    "types": "dist/src/index.d.ts",
-    "files": [
-      "dist"
-    ],  
-   "scripts": {
-     "test": "echo \"Error: no implemented\" && exit 1",
-     "build": "echo \"Error: no implemented\" && exit 1",
-     "lint": "echo \"Error: no implemented\" && exit 1",
-     "fix": "echo \"Error: no implemented\" && exit 1"
-   },
-   "keywords": [
-     "Energy",
-     "EnergyWeb",
-     "EW",
-     "DID",
-     "Registry",
-     "DID Registry"
-   ],
-   "homepage": "https://github.com/energywebfoundation/iam-roles/#readme",
-   "repository": {
-     "type": "git",
-     "url": "git+https://github.com/energywebfoundation/iam-roles.git"
-   },
-   "bugs": {
-     "url": "https://github.com/energywebfoundation/iam-roles/issues"
-   },
-   "author": {
-     "name": "EnergyWeb",
-     "url": "https://www.energyweb.org/"
-   },
-   "license": "GPL-3.0-or-later",
-   "devDependencies": {
-    "lerna": "4.0.0"
-   }
- }
\ No newline at end of file
+  "name": "@energyweb/iam-roles",
+  "version": "1.0.0",
+  "publishConfig": {
+    "access": "public",
```

Comment:
I thought `@energyweb/iam-roles` could be a bundled package with smaller individual package like `@energyweb/role-governance`. So someone can consume the entire module if needed, but this could be rare. Did I understand you correctly?